_components/input-mask/input-mask.md

```diff
@@ -0,0 +1,26 @@
+---
+category: Components
+component:
+  status: ready
+  package: usa-input-mask
+  dependencies: usa-input
+permalink: /components/input-mask/
+redirect_from:
+- /form-controls/11-text-input/
+title: Input mask
+type: component
+lead: An input mask is a string expression that constrains input to support valid input values.
+subnav:
+- text: Preview
+  href: '#input-mask-preview'
+- text: Code
+  href: '#input-mask-code'
+- text: Guidance
+  href: '#input-mask-guidance'
+- text: Package
+  href: '#input-mask-package'
```

Comment:
Not a make or break, but missing a link to the changelogs in the sidenav (Same for the other new components)

```suggestion
  href: '#input-mask-package'
- text: Latest updates
  href: '#input-mask-changelog'
```

Reply:
Totally forgot about the "other" sidenav